src/util-base64.h

```diff
@@ -32,7 +32,6 @@
 #define B64_BLOCK           4
 
 typedef enum {
-    BASE64_MODE_RELAX,
```

Comment:
will this not lead to a change in defaults? The removed enum has value 0, now rdc2045 has 0

Reply:
Indeed, there are other places where I left this first unused value of an enumeration...

Reply:
(there are also packed structures where apparently unused fields are kind of used)